packages/node/test/handlers.test.ts

```diff
@@ -209,6 +209,11 @@ describe('tracingHandler', () => {
     jest.restoreAllMocks();
   });
 
+  function getPropagationContext(): PropagationContext {
+    // @ts-expect-error accesing private property for test
+    return hub.getScope()._propagationContext;
```

Comment:
```suggestion
    return hub.getScope()['_propagationContext'];
```

You can access private properties without TS complaining and without `@ts-` comment this way - we are using this in replay tests quite a lot :)

Reply:
Oh TIL! good call.